entrypoint.sh

```diff
@@ -170,7 +170,7 @@ fi
 # Start docker service if needed (e.g. for docker-in-docker)
 if [[ ${_START_DOCKER_SERVICE} == "true" ]]; then
   echo "Starting docker service"
-  service docker start
+  sudo service docker start
```

Comment:
this container can be run as root, meaning `sudo` isnt necesary 100% of the time

Can you wrap it in `if [[ $(id -u) -eq 0 ]];` and `if [[ ${_RUN_AS_ROOT} == "true" ]]; then` logic for that edge case?